media-libs/vigra/vigra-1.11.1.recipe

```diff
@@ -0,0 +1,136 @@
+SUMMARY="A image processing and analysis library"
+DESCRIPTION="VIGRA stands for \"Vision with Generic Algorithms\". It's an \
+image processing and analysis library that puts its main emphasis on \
+customizable algorithms and data structures. VIGRA is especially strong for \
+multi-dimensional images, because many algorithms (e.g. filters, feature \
+computation, superpixels) are implemented for arbitrary high dimensions. \
+By using template techniques similar to those in the C++ Standard Template \
+Library, you can easily adapt any VIGRA component to the needs of your \
+application, without thereby giving up execution speed."
+HOMEPAGE="http://ukoethe.github.io/vigra/"
+COPYRIGHT="1998-2017 by Ullrich Koethe"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/ukoethe/vigra/archive/Version-1-11-1.tar.gz"
+CHECKSUM_SHA256="b2718250d28baf1932fcbe8e30f7e4d146e751ad0e726e375a72a0cdb4e3250e"
+SOURCE_FILENAME="$portName-$portVersion.tar.gz"
```

Comment:
```diff
-SOURCE_FILENAME="$portName-$portVersion.tar.gz"
+SOURCE_FILENAME="vigra-$portVersion.tar.gz"
```
(`$portName` = `vigra` when building on any primary arch...  and `vigra_x86` when building for x86 secondary arch.)

Reply:
Missed that one :) (left over from previous version, already removed a few instances for $portName there) :+1: